### PR TITLE
Fix imprecise statements for virtual hosting example.

### PR DESCRIPTION
**Pull request by SpecialYang**
In the provided ingress example, the traffic with other host value will be forwarded to service3. 
But the original doc is that only the traffic without host header can be forwarded to service3, which is not correct.
Moreover, the host header is required for request. When the target is ip address, the host value would be ip address. Otherwise, it should be a valid domain.

### CODE REVIEW

**Comment by sftim** [comment]
Hi. We recommend restricting fixes to a single localization. Would you be willing to revise this so it's just the English changes, @SpecialYang?

File: content/en/docs/concepts/policy/pod-security-policy.md [hunk 1]
```diff
@@ -11,7 +11,8 @@ weight: 30
 
 {{< feature-state for_k8s_version="v1.21" state="deprecated" >}}
 
-PodSecurityPolicy is deprecated as of Kubernetes v1.21, and will be removed in v1.25. For more information on the deprecation,
+PodSecurityPolicy is deprecated as of Kubernetes v1.21, and will be removed in v1.25. It has been replaced by
+[Pod Security Admission](/docs/concepts/security/pod-security-admission/). For more information on the deprecation,
```
Comment (sftim) [inline content/en/docs/concepts/policy/pod-security-policy.md:15]: This change looks like it might belong in a separate PR @SpecialYang ?

**Review by sftim — COMMENTED** [review]
/lgtm

File: content/en/docs/concepts/policy/pod-security-policy.md (hunk 1 above)
Comment (SpecialYang) [reply content/en/docs/concepts/policy/pod-security-policy.md]: This change has be merged already in repo. It appears here due to my mistake git op.

**Review by mehabhalodiya — COMMENTED** [review]
LGTM makes sense! 
Congrats on your first PR 🎉

**Comment by sftim** [comment]
/approve
Thanks @SpecialYang
